web/cypress/integration/hfi-calculator-page.spec.ts

```diff
@@ -32,26 +42,27 @@ describe('HFI Calculator Page', () => {
         fixture: 'hfi-calc/fire_centres.json'
       }).as('getFireCentres')
       cy.visit(HFI_CALC_ROUTE)
-      cy.selectFireCentreInDropdown('Kamloops')
       cy.wait('@getFireCentres')
+      cy.selectFireCentreInDropdown('Kamloops')
       cy.wait('@loadHFIResults')
     })
     it('save button should be disable', () => {
       // cypress/fixtures/hfi-calc/dailies-saved.json has "request_persist_success": true, save button should be looking at that.
       cy.getByTestId('save-button').should('be.disabled')
     })
-    it('fire start dropdown triggers hfi request', () => {
-      // Selecting a new fire start, should result in a new request to the server, that comes back with "request_persist_success": false, or
-      // no request_save, which should cause the save button to become enabled.
+    it('fire start dropdown triggers hfi set state request', () => {
+      // Selecting a new fire start, should result in a new request to the server, that comes back with "request_persist_success": true, or
```

Comment:
Please make this comment the test name

Reply:
`it('should send a new request to the server, that persists the change and sets the save button to disabled, when selecting a new fire start ')`

Reply:
@Sybrand , not addressed?

Reply:
Weird that it doesn't show as Outdated? I've pushed the change.